pkg/certificates/certificates.go

```diff
@@ -305,7 +305,24 @@ func (a *AuthorizationDB) UserIDs() ([]string, error) {
 	return keys.Strings(), nil
 }
 
-// Claim marks an authorization as claimed and records claim information
+// List returns all authorizations in the database.
+func (a *AuthorizationDB) List() (auths Authorizations, _ error) {
```

Comment:
why the blank identifier for this error? just curious

Reply:
@navillasa I wanted `auths` to be named so that I don't have to declare it in the method body but it's all or none with named return vars and since `err` is always to the left of `:=`, naming it would allocate an unused variable I think.